docs_src/microservices/device/supported/device-usb-camera/supplementary-info/advanced-options.md

```diff
@@ -1,5 +1,34 @@
 # Advanced Options
 
+## RTSP Auth
+The device service allows for rtsp stream authentication using the [rtsp-simple-server](https://github.com/aler9/mediamtx). Authentication is enabled by default.
+
+To configure the username and password for the authentication when buiding your own images, edit the fields in the 'configuration.yaml'
```

Comment:
Add that this only applies when using non-secure mode of EdgeX (and the non-secure not recommended warning of course)